WPMedia/fusion-news-theme-blocks
# THEMES-734: Numbered List Block | Resizer V2

## Description

This PR updates the numbered list block to use v2 of the resizer.

## Jira Ticket

- [THEMES-734](https://arcpublishing.atlassian.net/browse/THEMES-734)

## Acceptance Criteria

- Numbered List block is using image resizer v2

## Test Steps

1. Checkout this branch `git checkout THEMES-734-numbered-list-resizer-v2`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/numbered-list-block`
3. Add a numbered list block to a page and configure the content source.
4. Verify that the images load and are using resizer v2 URLs.

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
